sdk/modelsrepository/Azure.IoT.ModelsRepository/samples/readme.md

```diff
@@ -121,23 +123,23 @@ var client = new ModelsRepositoryClient();
 // When model dependency resolution is disabled, only the input dtmi(s) will be processed and
 // model dependencies (if any) will be ignored.
 var dtmi = "dtmi:com:example:TemperatureController;1";
-ModelResult result = await client.GetModelAsync(dtmi, ModelDependencyResolution.Disabled).ConfigureAwait(false);
+ModelResult result = await client.GetModelAsync(dtmi, ModelDependencyResolution.Disabled);
 
 // In this case the above dtmi has 2 model dependencies but are not returned
 // due to disabling model dependency resolution.
 Console.WriteLine($"{dtmi} resolved in {result.Content.Count} interfaces.");
 ```
 
-## Digital Twins Model Parser Integration
+## DTDL Parser Integration
```

Comment:
Header casing is inconsistent. Above it is Sentence case, but here it is Title Case. Could you standardize it in this document?